### PR TITLE
Feat: Adds exported_count; a reference to number of successful data e…

### PR DESCRIPTION
**Pull request by saivarunr**
…xports to the destination

**Fixes** # (*issue*)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by achettyiitr — APPROVED** [review]
LGTM

File: warehouse/upload.go [hunk 1]
```diff
@@ -2063,3 +2073,18 @@ func (job *UploadJobT) UpdateLocalSchema(schema warehouseutils.SchemaT) error {
 // 	}
 // 	return batchedStagingFiles
 // }
+
+func GetWarehouseDestinationSuccessCount(dbHandle *sql.DB, destinationID string) (successfulExportCount int, err error) {
+	query := fmt.Sprintf("select count(*) from %s where destination_id='%s' ", warehouseutils.WarehouseUploadsTable, destinationID)
+	rows, err := dbHandle.Query(query)
```
Comment (Jayachand) [inline warehouse/upload.go:2079]: If it is successful export count, query should contain `status = 'exported_data'`

File: warehouse/upload.go [hunk 2]
```diff
@@ -2063,3 +2073,18 @@ func (job *UploadJobT) UpdateLocalSchema(schema warehouseutils.SchemaT) error {
 // 	}
 // 	return batchedStagingFiles
 // }
+
+func GetWarehouseDestinationSuccessCount(dbHandle *sql.DB, destinationID string) (successfulExportCount int, err error) {
+	query := fmt.Sprintf("select count(*) from %s where destination_id='%s' ", warehouseutils.WarehouseUploadsTable, destinationID)
```
Comment (Jayachand) [inline warehouse/upload.go:2078]: Add source_id filter too to get indexed results

**Comment by achettyiitr** [comment]
Closing this as it is not required,

**Pull request closed without merging**